prod/westeurope/internal/api/functions_admin/function_app/terragrunt.hcl

```diff
@@ -88,6 +110,8 @@ inputs = {
       SERVICE_PRINCIPAL_CLIENT_ID = "ad-APPCLIENT-APIM-ID"
       SERVICE_PRINCIPAL_SECRET    = "ad-APPCLIENT-APIM-SECRET"
       SERVICE_PRINCIPAL_TENANT_ID = "common-AZURE-TENANT-ID"
+
+      PUBLIC_API_KEY               = "apim-IO-SERVICE-KEY"
```

Comment:
this needs to be changed with the private key of the gdpr service. we need to insert that one into keyvault first (ie. `apim-IO-GDPR-SERVICE-KEY`) @uolter